src/phoenix/server/api/input_types/ChatCompletionMessageInput.py

```diff
@@ -8,5 +11,9 @@
 class ChatCompletionMessageInput:
     role: ChatCompletionMessageRole
     content: JSON = strawberry.field(
-        description="The content of the message as JSON to support text and tools",
+        description="The content of the message as JSON to support various kinds of text"
     )
+    tool_calls: Optional[List[JSON]] = UNSET
+    """The tool calls that were made in the message"""
+    tool_call_id: Optional[str] = UNSET
+    """The ID of the tool call that was made in a prior message"""
```

Comment:
```suggestion
    """The ID of the tool call described in the current message"""
```

Reply:
this has to match the id of a tool call in a prior message, the wording might be a little confusing right now, but it needs to match a pre-existing tool call id, which would be message.tool_calls.id in a previous message

Reply:
I'll try to update a bit